pkg/unikontainers/unikernels/unikraft.go

```diff
@@ -83,6 +85,37 @@ func (u *Unikraft) Init(data UnikernelParams) error {
 	return nil
 }
 
+func (u *Unikraft) configureNet(ethDeviceIP, ethDeviceGateway, ethDeviceMask string) {
+	setCompatNetConfig := func() {
+		u.Net.Address = "netdev.ipv4_addr=" + ethDeviceIP
+		u.Net.Gateway = "netdev.ipv4_gw_addr=" + ethDeviceGateway
+		u.Net.Mask = "netdev.ipv4_subnet_mask=" + ethDeviceMask
+	}
+
+	setCurrentNetConfig := func() {
+		u.Net.Address = "netdev.ip=" + ethDeviceIP + "/24:" + ethDeviceGateway + ":8.8.8.8"
+	}
+
+	if u.Version == "" {
+		setCurrentNetConfig()
+		return
+	}
+
+	unikernelVersion, err := version.NewVersion(u.Version)
+	if err != nil {
+		setCurrentNetConfig()
+		return
+	}
+
+	targetVersion, _ := version.NewVersion(UnikraftCompatVersion)
```

Comment:
Should not we check for any errors here? Just for sanity?

Reply:
Yes, I added a check for this.